fil-proofs-tooling/scripts/benchy-remote.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+CMDS=$(cat <<EOF
+cd \$(mktemp -d)
+git clone -q https://github.com/filecoin-project/rust-fil-proofs.git
+cd rust-fil-proofs
+git checkout -q master
```

Comment:
what does `-q` do?

Reply:
It puts git in quiet mode, i.e. suppresses confirmation output